talk/app/talk-cli.hoon

```diff
@@ -3,43 +3,50 @@
 ::    pulls all known messages into a single stream.
 ::    type ;help for usage instructions.
 ::
-/-  chat
+/-  chat, cite, groups
 /+  shoe, default-agent, verb, dbug
 ::
 |%
 +$  card  card:shoe
 ::
-+$  versioned-state
-  $%  state-0
-  ==
-::
-+$  state-0
-  $:  %0
++$  state-1
+  $:  %1
       sessions=(map sole-id session)                ::  sole sessions
-      bound=(map flag:chat glyph)                   ::  bound chat glyphs
-      binds=(jug glyph flag:chat)                   ::  chat glyph lookup
+      bound=(map target glyph)                      ::  bound glyphs
+      binds=(jug glyph target)                      ::  glyph lookup
       settings=(set term)                           ::  frontend flags
       width=@ud                                     ::  display width
```

Comment:
```suggestion
      width=_80                                     ::  display width
```

This should be a per-session width, probably.

Reply:
Realized this would mean passing the session through +mr. Do you think this is fine considering width would be the only piece that uses the session here?